{
  "comments": [
    {
      "key": {
        "uuid": "33fa35b9_c9a71127",
        "filename": "/COMMIT_MSG",
        "patchSetId": 1
      },
      "lineNbr": 12,
      "author": {
        "id": 155
      },
      "writtenOn": "2021-05-05T13:00:04Z",
      "side": 1,
      "message": "Where does this error happen?",
      "range": {
        "startLine": 11,
        "startChar": 0,
        "endLine": 12,
        "endChar": 6
      },
      "revId": "a20a64ad17a1b294e5d4836df649ba30174b3110",
      "serverId": "270e2033-b340-4cff-9539-693957ebf0e7",
      "unresolved": true
    }
  ]
}